<ui version="4.0">
 <class>AnalyseSettingsDialog</class>
 <widget class="QDialog" name="AnalyseSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analisys settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Pre-Processing</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Filters</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QRadioButton" name="radioFilterNo">
             <property name="text">
              <string>No filter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioFilterYes">
             <property name="text">
              <string>Filter</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Analyse</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>Time analysis</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QCheckBox" name="checkTimeAnalisys">
             <property name="text">
              <string>Do time Analisys</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>Spectrum Analysis</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QCheckBox" name="checkSpectrumAnalysis">
             <property name="text">
              <string>Do spectrum analysis</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkWavelet">
             <property name="text">
              <string>Wavelet</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkLomb">
             <property name="text">
              <string>Lomb</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkFFT">
             <property name="text">
              <string>FFT (Fast Fourier transform)</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_8">
       <property name="title">
        <string>Post-Processing</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="text">
           <string>RadioButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string>RadioButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkSaveImage">
       <property name="text">
        <string>Save analyse images</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkSaveCSV">
       <property name="text">
        <string>Save analyse information to csv</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="regression">
       <property name="text">
        <string>Predict regression</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="markTraining">
       <property name="text">
        <string>Mark</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addInfo">
       <property name="text">
        <string>Add info</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startAnalyse">
       <property name="text">
        <string>Analyse</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
